2020-12-17 10:11:22,321 [1] INFO  EventService - Subscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [SiteId=2] [Machine=D126741] [Reason=RefreshConnect]
2020-12-17 10:16:02,436 [38] DEBUG EventQueue - Pushing event into queue: [Method:WorkPermitCreate] [DomainObject:WorkPermit] [Id:1754911] [Initiating Event Sink Uri:net.tcp://10.9.21.6:9771/]
2020-12-17 10:16:03,291 [22] DEBUG EventService - Should Notify Client of Event Called.  IsRelevant = False
2020-12-17 10:16:03,432 [22] DEBUG EventService - Should Notify Client of Event Called.  IsRelevant = True
2020-12-17 10:16:24,549 [22] ERROR EventService - Error calling proxy.An error has occurred while notifying a user event sink at net.tcp://10.0.0.210:9774/ of WorkPermitCreate with id: 1754911.  The error was: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ServiceModel.EndpointNotFoundException: Could not connect to net.tcp://10.0.0.210:9774/. The connection attempt lasted for a time span of 00:00:21.0280000. TCP error code 10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 10.0.0.210:9774.  ---> System.Net.Sockets.SocketException: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond 10.0.0.210:9774
   at System.Net.Sockets.Socket.DoConnect(EndPoint endPointSnapshot, SocketAddress socketAddress)
   at System.Net.Sockets.Socket.Connect(EndPoint remoteEP)
   at System.ServiceModel.Channels.SocketConnectionInitiator.Connect(Uri uri, TimeSpan timeout)
   --- End of inner exception stack trace ---

Server stack trace: 
   at System.ServiceModel.Channels.SocketConnectionInitiator.Connect(Uri uri, TimeSpan timeout)
   at System.ServiceModel.Channels.BufferedConnectionInitiator.Connect(Uri uri, TimeSpan timeout)
   at System.ServiceModel.Channels.ConnectionPoolHelper.EstablishConnection(TimeSpan timeout)
   at System.ServiceModel.Channels.ClientFramingDuplexSessionChannel.OnOpen(TimeSpan timeout)
   at System.ServiceModel.Channels.CommunicationObject.Open(TimeSpan timeout)
   at System.ServiceModel.Channels.DuplexSessionOneWayChannelFactory.DuplexSessionOutputChannel.OnSend(Message message, TimeSpan timeout)
   at System.ServiceModel.Channels.OutputChannel.Send(Message message, TimeSpan timeout)
   at System.ServiceModel.Dispatcher.OutputChannelBinder.Send(Message message, TimeSpan timeout)
   at System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   at System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   at System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   at System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   at Com.Suncor.Olt.Common.Services.IEventNotificationService.Notify(DomainEventArgs`1 e)
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Reflection.MethodBase.Invoke(Object obj, Object[] parameters)
   at Com.Suncor.Olt.Common.Wcf.ChannelInterceptor.Intercept(IInvocation invocation) in c:\code\windows\csc\OLT\trunk\src\Common\App\Wcf\ChannelInterceptor.cs:line 26
   at Castle.DynamicProxy.AbstractInvocation.Proceed()
   at Castle.Proxies.IEventNotificationServiceProxy.Notify(DomainEventArgs`1 e)
   at Com.Suncor.Olt.Remote.Services.EventService.NotifyClient(EventSink eventSink, DomainEventArgs`1 eventArgs) in c:\code\windows\csc\OLT\trunk\src\RemoteApp\App\Services\EventService.cs:line 263
2020-12-17 10:16:24,903 [22] DEBUG EventService - Should Notify Client of Event Called.  IsRelevant = False
2020-12-17 10:24:24,096 [39] DEBUG EventQueue - Pushing event into queue: [Method:WorkPermitUpdate] [DomainObject:WorkPermit] [Id:1754911] [Initiating Event Sink Uri:net.tcp://10.9.21.6:9771/]
2020-12-17 10:24:26,186 [22] DEBUG EventService - Should Notify Client of Event Called.  IsRelevant = False
2020-12-17 10:30:42,949 [39] INFO  EventService - Unsubscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [Reason=ApplicationExit]
2020-12-17 11:17:37,596 [23] INFO  EventService - Subscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [SiteId=13] [Machine=D126741] [Reason=RefreshConnect]
2020-12-17 15:30:40,306 [9] INFO  EventService - Unsubscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [Reason=ReconnectOnNetworkAvailabilityChange]
2020-12-17 15:30:40,731 [16] INFO  EventService - Subscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [SiteId=13] [Machine=D126741] [Reason=ReconnectOnNetworkAvailabilityChange]
2020-12-17 18:45:26,525 [9] INFO  EventService - Unsubscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [Reason=ReconnectOnNetworkAvailabilityChange]
2020-12-17 18:45:26,867 [5] INFO  EventService - Subscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [SiteId=13] [Machine=D126741] [Reason=ReconnectOnNetworkAvailabilityChange]
2020-12-17 20:00:05,420 [9] INFO  EventService - Unsubscribe: [ClientAddress=net.tcp://10.9.21.6:9771/] [Reason=ShiftGracePeriodEnd]
